db/migrate/20161206125911_create_admins.rb

```diff
@@ -0,0 +1,12 @@
+# add admin entity to database
+class CreateAdmins < ActiveRecord::Migration[5.0]
+  def change
+    create_table :admins do |t|
+      t.string :name
+      t.string :email, unique: true, null: false
```

Comment:
Add index for uniqueness, see http://stackoverflow.com/questions/1449459/how-to-make-column-unique-and-index-it-in-rails-migration